reggie/configs/data/west_virginia.yaml

```diff
@@ -93,72 +121,245 @@ column_names:
+  ZIP: text
+  MAIL HOUSE NO: text
+  MAIL STREET: text
+  MAIL STREET2: text
+  MAIL UNIT: text
+  MAIL CITY: text
+  MAIL STATE: text
+  MAIL ZIP: text
   REGISTRATION DATE: date
-  PartyAffiliation: character
-  Status: character
-  Congressional District: character
-  Senatorial District: character
-  Delegate District: character
-  Magisterial District: character
-  Precinct_Number: character
-  POLL_NAME: character
-
-county_codes:
-  Not_Provided: NP
+  PartyAffiliation: text
+  Status: text
+  Congressional District: text
+  Senatorial District: text
+  Delegate District: text
+  Magisterial District: text
+  Precinct_Number: text
+  POLL_NAME: text
+
+ordered_columns:
+  - ID_VOTER
+  - County_Name
+  - FIRST NAME
+  - Mid
+  - LAST NAME
+  - Suffix
+  - DATE OF BIRTH
+  - SEX
+  - HOUSE NO
+  - STREET
+  - STREET2
+  - UNIT
+  - CITY
+  - STATE
+  - ZIP
+  - MAIL HOUSE NO
+  - MAIL STREET
+  - MAIL STREET2
+  - MAIL UNIT
+  - MAIL CITY
+  - MAIL STATE
+  - MAIL ZIP
+  - REGISTRATION DATE
+  - PartyAffiliation
+  - Status
+  - Congressional District
+  - Senatorial District
+  - Delegate District
+  - Magisterial District
+  - Precinct_Number
+  - POLL_NAME
+
+# Modifications table
+##############################
+primary_key:
+  - pre_date
+  - post_date
+  - ID_VOTER
+  - change_type
+
+base_columns:
+  pre_date: timestamp
+  post_date: timestamp
+  ID_VOTER: text
+  change_type: text
+  pre_value: text
+  post_value: text
+  state: text
 
+# Summary table
+##############################
+summary_columns:
+  - date
+  - state
+  - locale
+  - totals
+  - male
+  - female
+  - ages_18_34
+  - ages_35_49
+  - ages_50_64
+  - ages_65_130
+  - unaffiliated
+  - democratic
+  - republican
+  - libertarian
+
+summary_types:
+  - timestamp
+  - text
+  - text
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+
+# Voter history
+##############################
+
+# id_voter|id_election|dt_election|Election_Name|cd_election_type|cd_election_cat|fl_absentee|fl_challenged|fl_early_voting|fl_regular
+hist_columns:
+  - id_voter
+  - id_election
+  - dt_election
+  - Election_Name
+  - cd_election_type
+  - cd_election_cat
+  - fl_absentee
+  - fl_challenged
+  - fl_early_voting
+  - fl_regular
+
+hist_columns_types:
+  id_voter: text
+  id_election: text
+  dt_election: date
+  Election_Name: text
+  cd_election_type: text
+  cd_election_cat: text
+  fl_absentee: text
+  fl_challenged: text
+  fl_early_voting: text
+  fl_regular: text
+
+# Voter history and other fields
+##############################
+generated_columns:
+  all_history: text[]
+  sparse_history: int[]
+  # absentee, early-voting, regular
+  votetype_history: text[]
+  # Challenged (boolean)
+  challenged_history: integer[]
+ordered_generated_columns:
+  - all_history
+  - sparse_history
+  - votetype_history
+  - challenged_history
+
+# Custom configuration
+# (legacy, unsure if needed yet)
+##############################

```

Comment:
yeah, these things (status_codes, gender_codes, race_codes, party_codes, num_party_codes) are not VoteShield legacy, this is something Delaney introduced in all her states. We don't need it, so if you want to take them out, feel free.